### PR TITLE
 Add muscle recovery table to workout overview page

### PR DESCRIPTION
**Pull request by thomasleese**
This works by finding the last exercise you did and it gets a percentage so that a muscle group is 100% recovered if it was last worked out over a week ago.

screen shot 2018-04-08 at 16 02 09

### CODE REVIEW

File: workout/models.py [hunk 1]
```diff
@@ -228,3 +222,50 @@ class ExerciseSet(models.Model):
...
+                    ).order_by('-start_time').values('start_time')[:1]
+                )
+            )
+        }
+
+    @property
+    def muscle_groups_workout_age(self):
+        now = timezone.now()
+
+        return {
```
Comment (vikashtank) [inline workout/models.py]: dictionary comprehension?

Comment (thomasleese) [reply workout/models.py]: Yeah, I think imma refactor this I don’t really like it

File: workout/templates/workout/dashboard.html [hunk 2]
```diff
@@ -14,6 +14,32 @@ <h1>Latest workout</h1>
...
+        <th scope="col">Muscle Group</th>
+        <th scope="col">Recovery</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {% for muscle_group, recovery in user.workout_profile.muscle_group_recovery.items %}
+        <tr>
+          <th scope="row">{{ muscle_group }}</th>
+          {% widthratio recovery 1 100 as percentage %}
```
Comment (vikashtank) [inline workout/templates/workout/dashboard.html]: widthratio???

File: workout/models.py (hunk 1 above)
Comment (thomasleese) [reply workout/models.py]: I've refactored this now!

**Pull request merged**